presto-spi/src/main/java/io/prestosql/spi/type/VarcharType.java

```diff
@@ -14,6 +14,7 @@
 package io.prestosql.spi.type;
 
 import io.airlift.slice.Slice;
+import io.airlift.slice.SliceUtf8;
```

Comment:
What about `CharType`?

If this doesn't work for chars, please add something like that
```java
@Override
public Optional<Range> getRange()
{
    // ....
    return Optional.empty();
}
```

Reply:
It does, but I haven't implemented it yet. It'll probably do it later as a separate improvement.

Reply:
sure.
BTW Is there anything special about chars?